beauty-consulting-app/app/assets/components/ProfileImage.js

```diff
@@ -56,22 +66,64 @@ const ProfileImage = ({ username }) => {
 		}
 	};
 
+	const fetchProfilePicture = async () => {
+		try {
+			const response = await api.get(`/account/${username}/photo`, {
+				responseType: "arraybuffer",
+			});
+
+			const base64 = Buffer.from(response.data, "binary").toString(
+				"base64"
+			);
+			const contentType = response.headers["content-type"];
+			const dataUri = `data:${contentType};base64,${base64}`;
+
+			userContext.updateUserContext({
+				profilePhoto: dataUri,
+			});
+		} catch (error) {
+			// console.error("Error fetching updated profile picture:", error);
+		}
+	};
+
```

Comment:
This function seems to only be called after a photo is uploaded. This means that when I logout and login, I do not see the photo I uploaded previously. Maybe call this in a useEffect for the component?

Reply:
Yeah, that's an issue as well. I will address this in the next sprint if that's okay.